helix-term/src/keymap.rs

```diff
@@ -591,10 +591,10 @@ impl Default for Keymaps {
                 "t" => align_view_top,
                 "b" => align_view_bottom,
                 "m" => align_view_middle,
-                "k" => scroll_up,
-                "j" => scroll_down,
-                "b" => page_up,
-                "f" => page_down,
+                "k" | "up" => scroll_up,
+                "j" | "down" => scroll_down,
+                "b" | "S-up" => page_up,
+                "f" | "S-down" => page_down,
```

Comment:
We're changing these too https://github.com/helix-editor/helix/pull/1040

Can you only keep the aliases for `k`/`j`?

Reply:
Done 👍

Reply:
Edited the `kepmap.md` also